ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# We don't want to use OpenShift-CI build cluster namespace
+unset NAMESPACE
+
+oc project openstack
+
+# Create clouds.yaml file to be used in further tests.
+mkdir -p ~/.config/openstack
+
+cat > ~/.config/openstack/clouds.yaml << EOF
+$(oc get cm openstack-config -n openstack -o json | jq -r '.data["clouds.yaml"]')
+EOF
+export OS_CLOUD=default
+KEYSTONE_SECRET_NAME=$(oc get keystoneapi keystone -o json | jq -r .spec.secret)
+KEYSTONE_PASSWD_SELECT=$(oc get keystoneapi keystone -o json | jq -r .spec.passwordSelectors.admin)
+OS_PASSWORD=$(oc get secret "${KEYSTONE_SECRET_NAME}" -o json | jq -r .data.${KEYSTONE_PASSWD_SELECT} | base64 -d)
+export OS_PASSWORD
+
+# Because tempestconf complain if we don't have the password in the clouds.yaml
+sed -i "/project_domain_name/ a \      password: ${OS_PASSWORD}" ~/.config/openstack/clouds.yaml
+
+# Configuring tempest
+
+mkdir ~/tempest
+pushd ~/tempest
+
+tempest init openshift
+
+pushd ~/tempest/openshift
+
+discover-tempest-config --os-cloud ${OS_CLOUD} --debug --create
+
+tempest run --regex 'tempest.api.identity.*.v3'
```

Comment:
if the exit code is non-zero here, you will not get the results below. Don't you need to surround with:
```
set +e
tempest run --regex 'tempest.api.identity.*.v3'
EXIT_CODE=$?
set -e
```